tests/test_iotedgedev.py

```diff
@@ -492,3 +496,17 @@ def test_push_modules_to_local_registry(prepare_solution_with_env):
             remove_docker_container("registry")
         if "registry" in get_all_docker_images():
             remove_docker_image("registry:2")
+
+
+# def test_docker_build_status_output():
```

Comment:
Why commented this block and also include the new lib?

Reply:
The test case could not pass since the log of docker build is not captured. But the major part of test case is finished. I want to preserve current work so I can continue from where I left.
I can move the test cases to another branch and just commit the features if it's not good to commit the WIP here.

Reply:
So the TODO should be added before the test definition to make it more clear.